server/src/controllers/usersController.ts

```diff
@@ -0,0 +1,59 @@
+import { Request, Response } from 'express';
+// import { userModel } from "../models/userModel";
+import { v2 as cloudinary } from "cloudinary";
+
```

Comment:
Can you delete some code which is doubled and commented out, cause I think it is good to have it cleaner from the beginning than clean up at the end

Reply:
there was no doubling, just the function saving to mongoose. But I agree it's better not to have code that is not being in use, removed all the comments.